src/main/java/org/opentripplanner/routing/core/vehicle_sharing/KickScooterDescription.java

```diff
@@ -7,11 +7,20 @@
 import org.opentripplanner.routing.core.TraverseMode;
 import org.opentripplanner.routing.edgetype.StreetEdge;
 
-import java.util.Objects;
-
 @EqualsAndHashCode(callSuper = true)
 public class KickScooterDescription extends BikePathVehicleDescription {
-    protected static final double MAX_SPEED_IN_METERS_PER_SECOND_ON_BIKEPATH = 15. * (10. / 36.);
+
+    /*
+     Kickscooters are not allowed on streets with speed limit >30km/h
+     */
+    public static final double MAX_EDGE_TRAVERSE_SPEED_LOWER_BOUND = 30. * (10. / 36.);
+
+    /*
+     We want to route kickscooters on bikepaths rather than on streets (if it is possible),
+     so we assume kickscooters are a bit faster on bikepaths than on streets.
+     */
+    protected static final double MAX_SPEED_IN_METERS_PER_SECOND_ON_BIKEPATH = 20. * (10. / 36.);
+    protected static final double MAX_SPEED_IN_METERS_PER_SECOND_ON_ROAD = 19. * (10. / 36.);
```

Comment:
It seems that these changes alter the paths returned in response to the requests for some integration tests: 

ResponseItineraryTypeBasicIT.testBusAndKickscooter:98 
Expected: iterable containing ["WALK+KICKSCOOTER", "WALK+TRANSIT", "WALK+TRANSIT"]
but: item 0: was "WALK+TRANSIT"

ResponseItineraryTypeBasicIT.testKickscooter:73 » Processing Error reading ent...

Reply:
It turns out that pedestrian and bike paths doesn't have proper max speed set (this field doesn't make sense on those paths obviously). Fixed